### PR TITLE
Add toggle video and toggle audio Zoom commands

### PR DESCRIPTION
**Pull request by lcardito**
## Description

Two new scripts to toggle video and microphone in the current Zoom meeting

## Type of change

- [x] New script command
- [ ] Bug fix
- [ ] Improvement of an existing script
- [ ] Documentation update
- [ ] Toolkit change
- [ ] Other (Specify)

## Screenshot
image

image

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/communication/zoom/mute-mic.applescript [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/osascript
+
+# Required parameters:
+# @raycast.author Luigi Cardito
+# @raycast.authorURL https://github.com/lcardito (credits Faris Aziz https://github.com/farisaziz12)
```
Comment (unnamedd) [inline commands/communication/zoom/mute-mic.applescript:5]:
```suggestion
# @raycast.author Luigi Cardito
# @raycast.authorURL https://github.com/lcardito
# @raycast.author Faris Aziz
# @raycast.authorURL https://github.com/farisaziz12
```

File: commands/communication/zoom/mute-mic.applescript [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# @raycast.schemaVersion 1
+# @raycast.title Mute microphone
```
Comment (unnamedd) [inline commands/communication/zoom/mute-mic.applescript:7]:
```suggestion
# @raycast.title Mute Microphone
```

File: commands/communication/zoom/mute-mic.applescript [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+# @raycast.mode silent
+# @raycast.packageName Zoom
+# @raycast.description Mutes your microphone in the current meetin
```
Comment (unnamedd) [inline commands/communication/zoom/mute-mic.applescript:10]:
```suggestion
# @raycast.description Mute your microphone in the current meeting
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
Just some changes need to be done not in the code but in the meta tags.

Thanks for your contribution @lcardito!

**Comment by lcardito** [comment]
Thanks @unnamedd for the quick review! Planning to add the same for toggling the video, shall I raise a different PR or add another script here?

**Comment by unnamedd** [comment]
> Thanks @unnamedd for the quick review! Planning to add the same for toggling the video, shall I raise a different PR or add another script here?

It is up to you, as it is part of the Zoom package, you can add it here and I will review as soon you submit. Just don't forget to update the PR's title!

**Comment by lcardito** [comment]
> > Thanks @unnamedd for the quick review! Planning to add the same for toggling the video, shall I raise a different PR or add another script here?
> 
> It is up to you, as it is part of the Zoom package, you can add it here and I will review as soon you submit. Just don't forget to update the PR's title!

Done! Let me how this looks! Thanks!

File: commands/communication/zoom/toggle-mic.applescript [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+# @raycast.author Luigi Cardito
+# @raycast.authorURL https://github.com/lcardito
+# @raycast.author Faris Aziz
+# @raycast.authorURL https://github.com/farisaziz12
+# @raycast.schemaVersion 1
+# @raycast.title Toggle Microphone
+# @raycast.mode silent
+# @raycast.packageName Zoom
+# @raycast.description Mute/Unmute your microphone in the current meeting
+# @raycast.needsConfirmation false
```
Comment (unnamedd) [inline commands/communication/zoom/toggle-mic.applescript:13]: You can remove this line here

```suggestion
```

File: commands/communication/zoom/toggle-video.applescript [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+# @raycast.author Luigi Cardito
+# @raycast.authorURL https://github.com/lcardito
+# @raycast.author Faris Aziz
+# @raycast.authorURL https://github.com/farisaziz12
+# @raycast.schemaVersion 1
+# @raycast.title Toggle Video
+# @raycast.mode silent
+# @raycast.packageName Zoom
+# @raycast.description Show/Hide your microphone in the current meeting
+# @raycast.needsConfirmation false
```
Comment (unnamedd) [inline commands/communication/zoom/toggle-video.applescript:13]: Remove this one as well

```suggestion
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
@lcardito just remove the two lines as I suggested and we will be good to go!

**Review by unnamedd — APPROVED** [review]
Looks good to me! 

Thanks for your changes @lcardito! 🤘

**Comment by lcardito** [comment]
> Looks good to me!
> 
> Thanks for your changes @lcardito! 🤘

Thank you!!

**Comment by RSO** [comment]
Thank you for your contributions!!! 🎉

**Pull request merged**